ocaml/xapi/quicktest.ml

```diff
@@ -877,6 +878,7 @@ let _ =
        (try
           maybe_run_test "encodings" Quicktest_encodings.run_from_within_quicktest;
           maybe_run_test "vm-memory-constraints" Quicktest_vm_memory_constraints.run_from_within_quicktest;
+          maybe_run_test "bvt" (fun () -> Quicktest_bv_tests.start s !rpc);
```

Comment:
Why are these called bvt?

Reply:
The tests check low-level behaviour, so they're essentially basic verification tests. The plan is to add similar tests for calls affecting fields like device_config, generalising the test beyond just other_config calls.